### PR TITLE
6.2.0-Release-Docs

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_6.2.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-52971 Inconsistent hash and signing of RPM files:** Some small number of RPM files did not usen the `sha256` hash for sigining. This has been corrected and all RPM packages on the distribution ISO are digest sha256 for Common Criteria.
+------
+- **I95-52994 Routers continue to request the conductor configuration:** Resolved an issue where a managed router continued to request the configuration from the conductor even after a validation or datamodel incompatibility issue.
+------
+- **I95-52999 Package-based Interactive Install does not recognize Mellanox CX6 LX (10/25G) interfaces:** Resolved a rare case where NICs with kernel drivers that exist in EL8 but not in EL7 will not be enumerated during Interactive ISO installation. 
+------
+- **95-53000 process highway disconnected messages caused by NIC driver bug:** The DPDK driver code for the Broadcom NICs contained a bug that caused the querying of the extended statistic to fail. The Broadcom NIC driver has been upgraded to resolve the issue.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:143]: Missing 'I' in Jira ID

File: docs/release_notes_128t_6.2.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-46537 Azure Mini-R Platform Support:** The Azure Mini-R is now a supported platform for the SSR software. 
+------
+- **I95-46671 VRF Route Leaking:** VRF Route Leaking allows the intentinoal sharing of route information across VRF instances. For a more detailed explanation and configuration information please refer to [VRF Route Leaking](config_vrf_route_leaking.md).
+------
+- **I95-47133 Local Configuration Override:** Administrators can now make configuration changes to a local router without access to the conductor, and prevent the conductor from overriding those changes. For more information about using this feature, see [Local Configuration Override](how_to_local_config_override.md).
+------
+- **I95-48174 Expand supported values for DHCP option:** DHCP option 43 is now a supported option, as well as a binary encoded-type (hex/byte) support. Valid examples are `0xabcdef` and `0x123456`.
+------
+- **I95-50071 Lenovo SR650 platform support as a 100G ports solution:** WAITING TO FINALIZE
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:72]: - [ ] TODO - complete this item

Recommendation:
I95-50071 Lenovo SR650 platform w/100G support: Two 2x100G Mellanox ConnectX-6 Dx NICs supported in SR650 platform.

File: docs/release_notes_128t_6.2.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-52703 Tenancy consideration in Application ID lookup:** Application Identification service lookups now consider the source tenant. This allows matches to be specific to certain networks/users. See [AppID and Tenancy](concepts_appid.md#appid-and-tenancy) for more information.
+
+### Resolved Issues 
+
+- **The following CVEs have been resolved in this release:** I95-52554, I95-52644, I95-52645, I95-52956.
+------
+- **I95-42466 Changing the physical linux address of an HA interface breaks the configuration:** Resolved an issue where moving a non-forwarding fabric HA sync device-interface from one PCI address to another PCI address would not properly clean up the team interface from the old PCI address.
+------
+- **I95-47838 SSR does not remove networks on stop:** Resolved an issue where the SSR does not remove networks on stop. This has been corrected and the SSR now will cleanup networks on container stop and create them on restart to optimize start/restart process.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:90]: Too many internal details on this one. Recommended alternative:
- **I95-47838 SSR strands internal interfaces when enabling and disabling IDP** Resolved an issue where the SSR does not remove internal network interfaces when IDP is disabled.

File: docs/release_notes_128t_6.2.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-51766 TX lockup detector not enabled for LAG/bonded interfaces:** The datapath lockup detection mechanism has been re-enabled to run on bond interfaces. 
+------
+- **I95-51801 The SSR is unable to see DHCP ACK for the DHCP Request sent by an EX4100:** Added an `authoritative` field for DHCP servers to enable/disable `authoritative` mode, which allows the server to send a NAK in response to unknown clients. This field is set to `true` by default.
+------
+- **I95-51864 Ethernet Over SVR (EoSVR) not working for multi-hop SVR scenarios:** When EoSVR traffic traverses over a dogleg path in a HA node topology, traffic failed to traverse the middle node. EoSVR packets are no longer incorrectly dropped when routed over an inter-node path when coming from an SVR path.
+------
+- **I95-51992 Multi-queue support for Bond interfaces:** Support for a bond `device-interface` to use multiple RX/TX queues has been added. 
+------
+- **I95-52018 Path returned contained ip-address with no escaping:** The Inspector path has been modified with the ability to escape.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:108]: I don't understand this description. This defect pertains to IDP, which was introduced in 6.0, but this fix was also delivered to 5.6 as well. The PR does not have a release notes section, unfortunately. Please reach out to the developer and understand the customer-facing impact.

File: docs/release_notes_128t_6.2.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-52113 Application Identification on the SSR runs at 100% CPU utilization:** Resolved an overrun bug that was causing the SSR to enter a loop when loading port ranges. This issue has been resolved.
+------
+- **I95-52147 Adding and deleting bond interfaces with the same name would leave the interface in a down state. This issue has been resolved.
+------
+- **I95-52158 Spoke is rejecting hub BFD packets, and peering is unable to come up over LTE:** In a corner case where the spoke private LTE IP changes before BFD is up and the public/hub-received IP stays the same, the hub gets stuck in the init state. 
+This issue has been resolved; the LTE IP change is now handled it as a source-nat change, where the flows and actions can be recreated with the updated LTE private IP.
+------
+- **I95-52178 Reset of SSR IDP does not turn it off:** Resolved an issue when performing a factory reset on SSR IDP did not turn it off properly.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:117]: Recommended rewording:

- **I95-52178 factory-reset of SSR config does not does not disable IDP:** Resolved an issue when performing a factory reset on SSR, IDP does not turn it off properly.

File: docs/release_notes_128t_6.2.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+------
+- **I95-53288 Fetching detailed bond info from the conductor for routers fails:** The conductor was incorrectly posting a `JSONDecodeError` when trying to parse bond information that was missing from the router response. This issue has been resolved.
+------
+- **I95-53321 Syslog datamodel is limited:** Added the following configurable syslog facility values `auth`, `authpriv`, `cron`, `daemon`, `kern`, `lpr`, `mail`, `news`, `syslog`, `user`, and `uucp`.
+-----
+- **I95-53344 Exception on device interface tear down terminates process:** Resolved a rare case where Highway process can terminate and core during config changes if there is an underlying exception to a device-interface on removal.
+------
+- **WAN-1323 Remove bootstrapper interfaces after Mist Onboarding:** The bridge interface used for bootstrapping in the default linux environment is now removed.
+------
+**WAN-1728 SSR1000 series Out of band management improvements:** The following improvements have been made to enhance the management functionality in Mist Managed environments:
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:175]: new feature?

File: docs/release_notes_128t_6.2.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+ - NTP sync
+ - MIST & conductor connectivity
+ - Other host services  
+ - Support for static IP address (including VLAN)
+ - Failover support between the management and WAN interface
+------
+- **WAN-1735 Set log levels on SSR:** Added new `log_level` and `log_category` fields in the `root_override` section of the intent config.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:182]: New FEature?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 8]
```diff
@@ -83,12 +83,14 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
 ### Resolved Issues 
 
-- **The following CVEs have been resolved in this release:** I95-52554, I95-52644, I95-52645, I95-52956.
+- **The following CVEs have been resolved in this release:** I95-52554, I95-52644, I95-52645, I95-52956, I95-53476, I95-53477, I95-53622.
 ------
 - **I95-42466 Changing the physical linux address of an HA interface breaks the configuration:** Resolved an issue where moving a non-forwarding fabric HA sync device-interface from one PCI address to another PCI address would not properly clean up the team interface from the old PCI address.
 ------
 - **I95-47838 SSR does not remove networks on stop:** Resolved an issue where the SSR does not remove networks on stop. This has been corrected and the SSR now will cleanup networks on container stop and create them on restart to optimize start/restart process. 
 ------
+- **I95-50539 Common Criteria - SSR not handling and logging Multicast packets correctly:** Per Common Criteria guidelines, the SSR now filters and drops packets if the source address is a multicast source IP address.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:92]: Drop "Common Criteria".

- **I95-50539 SSR not handling and logging Multicast packets correctly:** Per security best practices, the SSR now filters and drops packets if the source address is a multicast source IP address.

File: docs/release_notes_128t_6.2.md [hunk 9]
```diff
@@ -118,6 +120,8 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
 ------
 - **I95-52279 Bond interface configured with VRRP not receiving UDP traffic when LACP is enabled:** An issue where VRRP Virtual MACs from being silently dropped has been resolved. Packets with VRRP virtual dest MACs are now processed by the Bond PMD when using LACP.
 ------
+- **I95-52414 User seeing other user's FIB entries and messages:** Resolved an issue where `show fib` included entries that the current user did not have permission to view.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:123]: Suggested rewording:

- **I95-52414 RBAC not being honored for `show fib` output:** Resolved an issue where `show fib` included entries that the current user did not have permission to view.

File: docs/release_notes_128t_6.2.md [hunk 10]
```diff
@@ -126,6 +130,8 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
 ------
 - **I95-52547 Unable to set DHCP option 160:** Resolved an issue where DHCP option 160 was being treated as a standard option and was unavailable to be defined as an option. When it was set, it would prevent the DHCP server from starting. This has been corrected. 
 ------
+- **I95-52889 Highway crash caused by a false negative waypoint exhaustion check:** Waypoint ports reinitialization that is triggered by a false negative exhaustion check can lead to duplicate waypoints and reverse flows on two sessionsresulting in a highway crash. This issue has been resolved.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:133]: need space after "two sessions"

File: docs/release_notes_128t_6.2.md [hunk 11]
```diff
@@ -140,11 +146,11 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
...
 ------
-- **95-53000 process highway disconnected messages caused by NIC driver bug:** The DPDK driver code for the Broadcom NICs contained a bug that caused the querying of the extended statistic to fail. The Broadcom NIC driver has been upgraded to resolve the issue.
+- **I95-53000 process highway disconnected messages caused by NIC driver bug:** The DPDK driver code for the Broadcom NICs contained a bug that caused the querying of the extended statistic to fail. The Broadcom NIC driver has been upgraded to resolve the issue.
 ------
 - **I95-53002 NTP setup check fails on startup:** Resolved an issue in the NTP startup sequence, due to an incorrect path for the NTP configuration.
 ------ 
-- **I95-53105 Conductor to router API RBAC rules not being followed:** Resolved an issue where the user is getting elevated to admin on the managed router, thus returning more data than necessary.
+- **I95-53009 Common Criteria - RPM Verification:** Compliance verification for all ISO RPMs has been added.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:153]: Suggested rewording:

- **I95-53009 RPM signature verification missing for all artifacts:** Verification for all ISO RPMs has been added.

File: docs/release_notes_128t_6.2.md [hunk 12]
```diff
@@ -170,6 +176,10 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
 -----
 - **I95-53344 Exception on device interface tear down terminates process:** Resolved a rare case where Highway process can terminate and core during config changes if there is an underlying exception to a device-interface on removal.
 ------
+- **I95-53393 Common Criteria - Allow empty password attempts via SSH:** The SSR counts login attempts with an empty password as failed login attempts. These  contribute to locking a user account if they reach the threshold (the value configured in `configure authority password-policy deny`,) within a short time window.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:179]: Suggested rewording:

- **Empty password attempts not counting towards user lockout:** The SSR now counts login attempts with an empty password as failed login attempts. These contribute to locking a user account if they reach the threshold (the value configured in `configure authority password-policy deny`,) within a short time window.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 13]
```diff
@@ -69,7 +69,8 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 ------
 - **I95-48174 Expand supported values for DHCP option:** DHCP option 43 is now a supported option, as well as a binary encoded-type (hex/byte) support. Valid examples are `0xabcdef` and `0x123456`.
 ------
-- **I95-50071 Lenovo SR650 platform support as a 100G ports solution:** WAITING TO FINALIZE
+- **I95-50071 Lenovo SR650 platform support as a 100G ports solution: WAITING TO FINALIZE
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:72]: - [ ] Complete this item

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md [hunk 14]
```diff
@@ -195,7 +171,7 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
 ### Caveats
 
 - **I95-52426 Alerts not issued when decreasing the action type on an IDP custom rule definition:** In a case where a user is modifying a rule to **decrease** the action type to an `alert`, alerts for that vulnerability will not be reported. The attack will be allowed to pass through undetected. For example, if the action `close-tcp-connection` is downgraded to `alert`, the attacks will pass through undetected.
-
-
-
-
+------
+- **I95-53777 Multicast traffic not passing after HA Failover:** This feature is currently in Beta; High Availability with Multicast has not been fully tested and may not be fully functional. Drop or complete loss of traffic may be seen when the primary node resumes traffic after a node failure and failover.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:175]: Suggested rewording to match LAG caveats:
- **I95-53777 Multicast traffic not passing after HA Failover:** High Availability with Multicast is not fully supported. Drop or complete loss of traffic may be seen when the primary node resumes traffic after a node failure and failover.

File: docs/release_notes_128t_6.2.md [hunk 15]
```diff
@@ -69,22 +44,23 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
 - **I95-52406 Download MIBs from the GUI:** A button has been added to the GUI, in the Documentation pane of the About Page, to download the SNMP MIB definitions for SSR.
 ------
 - **I95-52703 Tenancy consideration in Application ID lookup:** Application Identification service lookups now consider the source tenant. This allows matches to be specific to certain networks/users. See [AppID and Tenancy](concepts_appid.md#appid-and-tenancy) for more information.
 
 ### Resolved Issues 
 
-- **The following CVEs have been resolved in this release:** I95-52554, I95-52644, I95-52645, I95-52956, I95-53476, I95-53477, I95-53622.
+- **The following CVEs have been resolved in this release:** CVE-2023-2828, CVE-2022-42896, CVE-2023-1281, CVE-2023-1829, CVE-2023-2124, CVE-2023-2194, CVE-2023-2235, CVE-2023-20569, CVE-2023-20593, CVE-2023-38802, CVE-2022-41974, CVE-2023-32360, CVE-2023-22045, CVE-2023-22049.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:63]: Per the internal email thread, I believe these need to follow a specific format with a brief description and CVSS.

File: docs/release_notes_128t_6.1.md [hunk 16]
```diff
@@ -161,7 +161,7 @@ This issue is actively being addressed, and will be resolved in an upcoming patc
 ------
 - **I95-10141 LLDP Support:** The LLDP mode and parameters allow users to configure the device interface to disable LLDP advertisements, set a `receive-only` mode, or enable sending and receiving LLDP packets. For information about configuring LLDP, see [`lldp`](config_reference_guide.md#lldp).
 ------
-- **I95-20864 Support for Multicast:** Multicast is a “one source, many destinations” method of traffic distribution. For more information, see [Multicast](config_multicast.md). The previous implementation of multicast has been replaced by this new version and is no longer supported. The issue relating to encryption (I95-48792) is addressed in the new implementation. 
+- **I95-20864 Support for Multicast:** Multicast is a “one source, many destinations” method of traffic distribution. For more information, see [Multicast](config_multicast.md). The previous implementation of multicast has been replaced by this new version and is no longer supported. The issue relating to encryption (I95-48792) is addressed in the new implementation. Note that this feature is currently in Beta; refer to the Caveats section for additional information.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:164]: I don't recall Multicast being in Beta.

Comment (MichaelBaj) [reply docs/release_notes_128t_6.1.md:164]: Remove newly added sentence.

File: docs/release_notes_128t_6.1.md [hunk 17]
```diff
@@ -268,9 +268,11 @@ This issue is actively being addressed, and will be resolved in an upcoming patc
...
+  **Example:** If `HTTP:SQL:INJ:HEADER-1` is excluded from the ruleset, some other `HTTP` attacks may pass through undetected. **This behavior only occurs when decreasing the action type in the rule;** i.e.; the action `close-tcp-connection` is downgraded to `alert`. 
 
-This issue is actively being addressed, and will be resolved in an upcoming patch release. If you need to use this specific functionality, we recommend avoiding this configuration and waiting for the SSR 6.1.5 patch release. 
+  This issue is actively being addressed, and will be resolved in an upcoming patch release. If you need to use this specific functionality, we recommend avoiding this configuration and waiting for the SSR 6.1.5 patch release.
+------
+- **I95-53777 Multicast traffic not passing after HA Failover:** This feature is currently in Beta; High Availability with Multicast has not been fully tested and may not be fully functional. Drop or complete loss of traffic may be seen when the primary node resumes traffic after a node failure and failover.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.1.md:275]: See my wording in the 6.2 release notes section.

File: docs/release_notes_128t_6.2.md [hunk 18]
```diff
@@ -69,22 +44,23 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+- **I95-51194 Support for Multicast Source Discovery Protocol (MSDP), and the SSR as a Rendezvous Point (RP):** MSDP support has been added, allowing the RP's to share active Multicast Sources. For detailed configuration information, see [Multicast Source Discovery Protocol (MSDP)](config_multicast.md#multicast-source-discovery-protocol-msdp). 
 ------
 - **I95-51450 Support for 100/Full Speed/Duplex on Intel I225-V Driver NICs:** The DPDK driver has been updated to allow fixed speed and duplex configuration to work with IGC i225 NICs.
 ------
+- **I95-51544 Add the option to use the physical source MAC on VRRP interfaces:** The coommand [`use-physical-address`](config_command_guide.md#configure-authority-router-node-device-interface-vrrp-use-physical-address) has been added to the device-interface configuration. This allows VRRP to use the physical MAC rather than the virtual MAC and prevents platforms that reject the virtual MAC from dropping traffic.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:55]: Typo "coomand"

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.2.md (hunk 15 above)
Comment (MichaelBaj) [reply docs/release_notes_128t_6.2.md:63]: Looks like a list of CVEs is sufficient. Let's start with this approach. Thank you.

File: docs/release_notes_128t_6.2.md (hunk 18 above)
Comment (Bhautik) [reply docs/release_notes_128t_6.2.md:55]: Release version is 6.2.0-39 and date is November 16th

**Review by Bhautik — APPROVED** [review]

File: docs/release_notes_128t_6.2.md [hunk 19]
```diff
@@ -155,8 +157,22 @@ This issue has been resolved; the LTE IP change is now handled it as a source-na
...
 ------
+- **I95-53472 Service Routes passing validation on conductor but then failing on local router:** The validation process on the conductor has been updated to identify service-routes with deleted or empty destination lists as invalid.  
+------
+- **I95-53538 Custom audit rules not preserved on SSR upgrade:** Resolved an issue where the image-based upgrade (IBU) was not preserving audit rules or `dnf.conf`.
+------
 - **I95-53583 `show service-path` on hub always shows ADDR_UNKNOWN IP address for the peers:** Add remote host name to `show service-path [detail]` . This resolves an issue where hub routers interacting with dynamic spoke interfaces do not show information in `show service-path detail` to identify those individual paths. 
 ------
+- **I95-53641 BGP between peers does not immediately transition to the Connected state:** A change has been made to reduce the time that BGP routes learned from a BGP over SVR neighbor are withdrawn when we lose all peer paths to the neighbor.
+------
+- **I95-53894 DNS cache-service does not start:** Resolved an issue that resulted in a stack trace in the logs which starts with message `No TimeoutQueue:`.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.2.md:168]: The release notes section in the PR is not accurate.
Resolved a race condition causes the dns process to fail to start. The log message `No TimeoutQueue:` can be seen in the logs during this condition.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md [hunk 20]
```diff
@@ -79,6 +79,8 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
 
 - **The following CVE's have been identified and addressed in this release:** CVE-2021-44531, CVE-2021-44532, CVE-2021-44533, CVE-2021-44906, CVE-2022-35256, CVE-2022-21824, CVE-2022-28733, CVE-2022-2964, CVE-2022-4139, CVE-2023-22809, CVE-2023-21830, CVE-2023-21843, CVE-2022-4254, CVE-2021-25220 CVE-2022-2795, CVE-2022-38023, CVE-2022-37434, CVE-2023-0286, CVE-2023-0767, CVE-2023-0286, I95-50535, CVE-2023-23918, CVE-2023-23920.
 ------
+- **The following issues have been identified during penetration testing and addressed in this release:** I95-48445, I95-48448, I95-49456, I95-49912, I95-49913, I95-49914.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md:82]: I have called out the issues that were not identified in Pen testing; the two that remain look to be significant and if given public visibility could be bad.

File: docs/release_notes_128t_6.0.md [hunk 21]
```diff
@@ -114,6 +111,12 @@ Please refer to [Password Policies](config_password_policies.md) for updated pas
 ------
 - **I95-49791 Add audit rules to track modification of grub config files:** Added rules to log notifications in case of changes to grub configuration files.
 ------
+- **I95-49912 Login banner not displayed on serial console:** The login banner is now displayed on the serial console.
+------
+- **I95-49913 Some Login/Logout events not logged in Audit Logs:** All login and logout events are now logged in the audit logs.
+------
+- **I95-49914 List of cryptographic algorithms does not match:** A discrepancy between algorithms claimed and those offered was found. This issue has been addressed and the list of claimed and offered algorithms is now complete.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:118]: This is marked as Exclude from Release Notes. Please remove.

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md (hunk 21 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md]: done

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**